extensions/common/iam/identity-trust/identity-trust-issuers-configuration/src/main/java/org/eclipse/edc/identitytrust/issuer/configuration/TrustedIssuerConfigurationExtension.java

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.identitytrust.issuer.configuration;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.eclipse.edc.identitytrust.TrustedIssuerRegistry;
+import org.eclipse.edc.identitytrust.model.Issuer;
+import org.eclipse.edc.runtime.metamodel.annotation.Extension;
+import org.eclipse.edc.runtime.metamodel.annotation.Inject;
+import org.eclipse.edc.spi.EdcException;
+import org.eclipse.edc.spi.system.ServiceExtension;
+import org.eclipse.edc.spi.system.ServiceExtensionContext;
+import org.eclipse.edc.spi.system.configuration.Config;
+import org.eclipse.edc.spi.types.TypeManager;
+
+import java.util.Map;
+
+import static org.eclipse.edc.identitytrust.issuer.configuration.TrustedIssuerConfigurationExtension.NAME;
+
+@Extension(NAME)
+public class TrustedIssuerConfigurationExtension implements ServiceExtension {
+
+    public static final String CONFIG_PREFIX = "edc.iam.trusted-issuer";
+    public static final String PROPERTIES_SUFFIX = "properties";
+    public static final String ID_SUFFIX = "id";
+    protected static final String NAME = "Trusted Issuers Configuration Extensions";
+
+    @Inject
+    private TrustedIssuerRegistry trustedIssuerRegistry;
+    @Inject
+    private TypeManager typeManager;
+
+    @Override
+    public void initialize(ServiceExtensionContext context) {
+        var config = context.getConfig(CONFIG_PREFIX);
+        var issuers = config.partition().map(this::configureIssuer).toList();
+        if (issuers.isEmpty()) {
+            throw new EdcException("The list of trusted issuers is empty");
```

Comment:
Should this abort the runtime or just issue an error?

Reply:
I though of just issuing an error message but then in theory it will not work anyway at runtime since the list is empty

Reply:
Yeah, I thought though it may be "nicer" to not abort, but don't have a strong opinion.